corehq/motech/fhir/signals.py

```diff
@@ -0,0 +1,11 @@
+from couchforms.signals import successful_form_received
+
+from corehq.motech.repeaters.signals import create_repeat_records
+
+
+def create_fhir_repeat_records(sender, xform, **kwargs):
+    from .repeaters import FHIRRepeater
+    create_repeat_records(FHIRRepeater, xform)
+
+
+successful_form_received.connect(create_fhir_repeat_records)
```

Comment:
I think you probably want a `dispatch_uid` here to avoid duplicate calls:
https://docs.djangoproject.com/en/3.0/topics/signals/#preventing-duplicate-signals

Sounds like it's usually not an issue, but I think that's a good practice we should get in the habit of following.